lib/components/layout/virtruLayout.js

```diff
@@ -5,9 +5,9 @@ import cn from 'classnames';
 import { Footer } from '@';
 import styles from './virtruLayout.css';
 
-const VirtruLayout = ({ header, sider, content, className }) => {
+const VirtruLayout = ({ header, sider, content, className, ...props }) => {
```

Comment:
What exactly do we need here? I think we should specify the props so that when we look at this component we know what it actually accepts.

Reply:
I want to be able to pass props for the div element like styles

Reply:
There really shouldn't be one-off styles when using the design system.So any style that you would put in the `style=` should be implemented as maybe a `type` of `Layout` or something. So others can get the same styling and apps will be consistent.

Reply:
I definitely agree with you related to styles for this component but in my case, I'm going to provide a `style` prop to add CSS variables to this main wrapper in Secure Share. It'll look like this:
```
<VirtruLayout style={{ '--vds-color-blue-primary': theme.promaryColor }} />
```

Reply:
I see, then I would still suggest a defined prop. Can we use a `primaryColor` prop (Or better name)?